app/src/main/java/com/ntg/user/mvpsample/MainActivity.java

```diff
@@ -1,32 +1,101 @@
 package com.ntg.user.mvpsample;
 
+import android.content.Context;
+import android.content.DialogInterface;
 import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
-import android.view.Menu;
-import android.view.MenuItem;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.EditText;
 import android.widget.Toast;
 
-public class MainActivity extends AppCompatActivity {
 
+import com.ntg.user.mvpsample.remote.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity implements TasksContract.View {
+    TaskAdapter adapter;
+    TasksRepo tasksRepo;
+    TasksContract.Presenter presenter;
+    RecyclerView recyclerView;
+    private FloatingActionButton btn;
+    EditText taskName, taskDesc;
+    Context context = this;
+    
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        recyclerView = findViewById(R.id.recyclerView);
+        btn = findViewById(R.id.fab);
+        tasksRepo = new TasksRepo();
+        presenter = new TaskPresenter(this, tasksRepo);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+        adapter = new TaskAdapter(new ArrayList<Task>(0));
+        recyclerView.setAdapter(adapter);
+        presenter.getTask();
+        // add btn listener
+        btn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View arg0) {
+                // get prompts.xml view
```

Comment:
remove any comment inside block of code , your code has to readable enough so that no need for comments
just add java doc for class /methods